src/V3Options.cpp

```diff
@@ -111,6 +111,26 @@ V3LangCode::V3LangCode(const char* textp) {
     m_e = V3LangCode::L_ERROR;
 }
 
+//######################################################################
+// V3HierarchyBlockOption class functions
+
+V3HierarchyBlockOption::V3HierarchyBlockOption(const std::string& opts) {
+    std::vector<std::string> vals;
+    for (std::string::size_type s = 0; s != opts.npos && s < opts.size(); ) {
+        const std::string::size_type delim_or_end = opts.find(',', s);
+        vals.push_back(opts.substr(s, delim_or_end));
+        if ( delim_or_end == opts.npos) break;
+        s = delim_or_end + 1;
+    }
+    UASSERT(vals.size() >= 2, "at least two entries (origName and mangledName) are necessary");
+    m_origName = vals[0];
+    m_mangledName = vals[1];
+    //TODO  set m_parameters
```

Comment:
//FIXME set m_parameters

The FIXME will fail the build's fixme test so we don't forget to clean it up.